src/nft/utils/scrollToTop.ts

```diff
@@ -0,0 +1,3 @@
+const DESKTOP_OFFSET = 420
```

Comment:
what does this desktop offset refer to?

Reply:
I think its the distance from the top window to the filters bar - when all the components are added in I think it should add up to that according to the figma

Reply:
i feel like that's a bit brittle, if anything up there changes then this no longer is correct?

Reply:
yeah I agree let me remove this for now to get this merged in. Later I guess we can build something that actually scrolls to the top of the container + the height of the filters bar